app/helpers/application_helper.rb

```diff
@@ -191,9 +191,18 @@ def flash_div *keys
     keys.collect { |key|
       if flash[key]
         if flash[key].is_a?(Array)
-          content_tag(:div, content_tag(:ul, flash[key].map { |flash_item| content_tag(:li, h(flash_item)) }.join("\n").html_safe), class: "flash #{key}")
+          content_tag(
+            :div,
+            content_tag(
+              :ul,
+              flash[key].map do |flash_item|
+                content_tag(:li, sanitize(flash_item))
+              end.join("\n").html_safe
```

Comment:
I think we can use [`safe_join`](https://apidock.com/rails/v6.1.3.1/ActionView/Helpers/OutputSafetyHelper/safe_join) to avoid html_safe.

Reply:
Done!